ts/Dashboards/Layout/Layout.ts

```diff
@@ -274,9 +274,9 @@ class Layout extends GUIElement {
         const layout = this;
 
         // Destroy rows.
-        for (let i = 0, iEnd = layout.rows.length; i < iEnd; ++i) {
-            layout.rows[i].destroy();
-        }
+        layout.rows.forEach(function (row): void {
+            row.destroy();
+        });
```

Comment:
It is slower. Please restore the regular loop.

Reply:
I agree that the regular for (in the appropriate declaration - see in other components) is the fastest.